README.md

```diff
@@ -5,4 +5,16 @@
 [![Codacy Badge](https://api.codacy.com/project/badge/Grade/d1d73fb87e084904993f968178274835)](https://www.codacy.com/gh/ie3-institute/PowerSystemDataModel?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=ie3-institute/PowerSystemDatamodel&amp;utm_campaign=Badge_Grade)
 [![License](https://img.shields.io/github/license/ie3-institute/powersystemdatamodel)](https://github.com/ie3-institute/powersystemdatamodel/blob/master/LICENSE)
 
-Provides an elaborated data model to model energy systems with a high granularity e.g. for bottom-up simulations. Additionally, useful functionalities to handle the model i/o is provided as well.
+Provides an extensive data model capable of modelling energy systems with high granularity e.g. for bottom-up simulations. Additionally, useful functions to process, augment and furnish model i/o information is provided. Effective handling of geographic information related to power grids is also possible.
+
+## Supported physical models:
+      - Power Grids containing nodes, lines, swithces and transformers
+      - Conventional and renewable generating components such as fixed feed, biomass plants, wind turbines and photovoltaics
+      - Power to heat units e.g. combined heat and power plants and heat pumps
+      - Electrical storages, electric vehicles and charging stations
+      - Thermal units consisting of thermal building and cylindrical storage models
+
+## Supported simulation data:
+      Load profiles, weather data etc.
+      
+Data is represented through CSV files
```

Comment:
That doesn't hold true / is not necessary. *For now* we only provide I/O capabilities for .csv files, but this can easily be extended in the future. Furthermore we *do not* provide data, but only the model. Hence, I would propose removing this line

Reply:
Modified